<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EI1MF8H" failures="1" tests="7" name="Test Suite" time="54.542" errors="0" timestamp="2022-04-11T17:51:22 BDT">
  <testcase classname="normalUser.LoginTest" name="verifyElemntsOnPageTest" time="0.477"/>
  <testcase classname="normalUser.LoginTest" name="verifyLogin" time="5.595">
    <failure type="java.lang.AssertionError" message="expected [https://dev.zntral.net/dashboarda] but found [https://dev.zntral.net/dashboard]">
      <![CDATA[java.lang.AssertionError: expected [https://dev.zntral.net/dashboarda] but found [https://dev.zntral.net/dashboard]
at normalUser.LoginTest.verifyLogin(LoginTest.java:113)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
  <testcase classname="normalUser.LoginTest" name="loginWithoutUsername" time="1.556"/>
  <testcase classname="normalUser.LoginTest" name="loginWithoutPassword" time="1.699"/>
  <testcase classname="normalUser.LoginTest" name="loginWithEmptyField" time="0.259"/>
  <testcase classname="normalUser.LoginTest" name="forgotPassword" time="3.426"/>
  <testcase classname="normalUser.LoginTest" name="randomInput" time="5.601"/>
</testsuite> <!-- Test Suite -->
